v20.2/manage-a-backup-schedule.md

```diff
@@ -0,0 +1,222 @@
+---
+title: Manage a Backup Schedule
+summary: Learn about how to create, manage, and restore from a schedule of periodic backups.
+toc: true
+---
+
+<span class="version-tag">New in v20.2:</span> You can create schedules in CockroachDB for periodic backups. Once a [backup schedule is created](#create-a-new-backup-schedule), you can do the following:
+
+- [Set up monitoring for the backup schedule](#set-up-monitoring-for-the-backup-schedule)
+- [View and control the backup schedule](#view-and-control-the-backup-schedule)
+- [View and control a backup initiated by a schedule](#view-and-control-a-backup-initiated-by-a-schedule)
+
+## Create a new backup schedule
+
+{% include copy-clipboard.html %}
+~~~
+> CREATE SCHEDULE schedule_name
```

Comment:
schedule_name is optional, btw.
If not given, the schedule will be assigned a name "BACKUP <some number>"

Reply:
Added this info the the `CREATE SCHEDULE FOR BACKUP` page > `label` description.

Reply:
We might also want to call it schedule_label to avoid confusion here. We should use a consistent term throughout the docs IMO